### ISSUE TITLE
[plugin_manager] - Can't logon while trying to translate plugin

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#326]
```
Token: 8a063e2d1fae8802fd5dd853dbbc2b41 (Don't modify this token!)
Version: 2.7.3 (06-04-2012)
PHP: 5.4.0
RCMAIL: 0.7.2
Database: mysql
SERVER: Microsoft-IIS/7.5
----
I.  Issue Description:
I can't logon to the translate website if I click on translate plugin. I get a 
screen with a prefilled logon name (that one is ok) and an prefilled server 
name (%d:143, does not seem right) that can't be changed. See attachment. Looks 
like %d:143 is the culprit that keeps me from a succesful logon.

II. Steps to reproduce the Issue:
1. Go to plugin manager
2. Choose a plugin
3. Click on x% (translate this plugin)

```

Original issue reported on code.google.com by `hijack...@gmail.com` on 14 Apr 2012 at 10:05

Attachments:
- [online_translation.jpg](https://storage.googleapis.com/google-code-attachments/myroundcube/issue-326/comment-0/online_translation.jpg)

**Comment by GoogleCodeExporter** [comment]
```
I have resetted the password. It is now the same as for your calendar account 
(jackcal). It works for me. Perhaps you used a wrong password?
```

Original comment by `rol...@liebl.ath.cx` on 15 Apr 2012 at 4:15
- Changed state: **Reproducing**

**Comment by GoogleCodeExporter** [comment]
```
Should be fixed in plugin_manager 2.7.4. You should be able to login into real 
time translation mode from your own Roundcube Plugin Manager Site by using 
hi***@gmail.com.
```

Original comment by `rol...@liebl.ath.cx` on 15 Apr 2012 at 4:54

**Comment by GoogleCodeExporter** [comment]
```
Unfortunately plugin_manager 2.7.4 doesn't fix my issue. However I got it 
working by hardcoding my logon account in file plugin_manager.php in the 
following way:

Line 648:
Change 
'&_translator=' . $user . '&_host=' . $host .
into
'&_translator=' . "hi***@mail4us.net" . '&_host=' . "localhost" .     
--> hi***@gmail.com doesn't work as you suggested in comment 2.


So line 648 reads now:
          $translation = html::tag('td', array('align' => 'right', 'title' => $plugin . ' :: ' . $this->gettext('translate') . '...'), html::tag('a', array('href' => $this->mirror . '?_action=plugin.plugin_server_translate&_hl=' . $_SESSION['language'] . '&_plugin=' . $plugin . '&_translator=' . "hi***@mail4us.net" . '&_host=' . "localhost" . '&_port=' . $this->rcmail->config->get('default_port'), 'target' => '_new'), ($mirror[$plugin]['lc'] * 100)) . ' %');


```

Original comment by `hijack...@gmail.com` on 15 Apr 2012 at 9:55

**Comment by GoogleCodeExporter** [comment]
```
I'd like to get this working. Any chance to FTP into your installation?
```

Original comment by `rol...@liebl.ath.cx` on 15 Apr 2012 at 11:07

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Before I'm trying to set up a FTP server I like you to know the following.

Hardcoding my mail4us.net logon account didn't do the trick, probably I was 
using a cache or something.

What did I do next:
- Using plugin_manager.php version 2.7.4 (no modifications anymore, just the 
original version)
- Go to Plugin Manager Center (my local installation)
- Click on x% to try to translate a plugin

* Expected URL (something like): 
http://mirror.mail4us.net/?_action=plugin.plugin_server_translate&_hl=nl_NL&_plu
gin=plugin_manager&_translator=hi***@mail4us.net&_host=localhost&_port=143

* Instead I'm routed to: 
http://mirror.mail4us.net/translator/index.php
This is the location where you're asked to save the localization file and mail 
it to myroundcube|at|mail4us.net.

```

Original comment by `hijack...@gmail.com` on 15 Apr 2012 at 12:58

**Comment by GoogleCodeExporter** [comment]
```
Thanks for granting me remote access. Fixed in plugin_manager v2.7.5.
```

Original comment by `rol...@liebl.ath.cx` on 18 Apr 2012 at 4:04
- Changed state: **Fixed**
